client/landing/gutenboarding/onboarding-block/acquire-intent/style.scss

```diff
@@ -167,7 +167,6 @@
 		height: 100%;
 		left: 0;
 		bottom: 0;
-		z-index: 2;
```

Comment:
Now, when wrapped to a new line, the placeholder area (marked with red) is no longer focusable with click.
<img width="859" alt="Screenshot 2020-05-19 at 12 00 05" src="https://user-images.githubusercontent.com/14192054/82308544-9d510e80-99ca-11ea-8987-83142e28ee27.png">